src/extensions/vendor/vendor.md

```diff
@@ -0,0 +1,24 @@
+---
+group: extensions
+title: Vendor Bundled Extensions (VBEs)
+permalink: extensions/vendor/
+---
+
+Vendor Bundled Extensions (VBEs) are third-party extensions that we have thoroughly tested and included with each supported version of {{site.data.var.ce}} and {{site.data.var.ee}}.
+
+Current VBEs include:
+
+-  (Amazon Pay)[https://docs.magento.com/m2/ce/user_guide/payment/amazon-pay.html]
```

Comment:
should this be `[text](link)` instead?

Reply:
Were there strong reasons not to use recommended style with link definitions?

Reply:
thanks @dshevtsov, I got the syntax wrong.

no, there aren't _strong_ reasons for not using the link definition convention. i personally find it less usable as a maintainer than inlining them. since our style guide doesn't disallow inlining, i think i'll leave it as-is.